### ISSUE TITLE
Nodes missing CNC and circular saw support

### ISSUE THREAD

**Issue by OgelGames** [pandorabox-io/pandorabox.io#491]
* `planet_mars` black, blue, and red marble (also polished and brick varients)
* `moreores` silver and mithril blocks

**Comment by BuckarooBanzay** [comment]
> planet_mars black, blue, and red marble (also polished and brick varients)

I didn't get around to those but they were planned someday... :stuck_out_tongue: 

> some nodes from scifi_nodes

I should still have access to that repo, changing things there should go fairly fast...

**Comment by BobFred7** [comment]
There are a bunch of things from scifi_nodes that should be circular-saw-able but aren't, the first thing that comes to mind is octagon glass

**Comment by S-S-X** [comment]
added lead block to first comment

**Comment by BuckarooBanzay** [comment]
I've updated the top comment and separated between CNC and circular saw.
We currently have 21038 registered nodes, just to keep the limit of 32k in mind (https://monitoring.minetest.land/d/QcXQ62fZk/registered-counts?orgId=1&refresh=5s)

I see if i can get to the mars marbles later today, no promises though... :stuck_out_tongue:

**Comment by OgelGames** [comment]
> We currently have 21038 registered nodes, just to keep the limit of 32k in mind

Yeah, that's something I was concerned about, considering each node added to the circular saw adds 48 more nodes, and the CNC adds 21 (I think), that would be an extra 237 just for all the marble nodes...

**Comment by S-S-X** [comment]
Somewhat related idea (up to you how related it actually is...): https://github.com/mt-mods/technic/issues/185
